themes/default/layouts/migrate/terraform.html

```diff
@@ -3,7 +3,10 @@
 {{ end }}
 
 {{ define "main" }}
-    <section id="overview" class="container mx-auto px-2 my-16">
+    <section id="overview" class="container mx-auto px-2 my-4">
+        <div class="flex justify-center rounded-md bg-violet-100">
+            <p class="text-center font-bold">&#x2728; <a href="#migrate">[NEW] Contact us and we'll migrate your Terraform to a modern language</a> &#x2728;</p>
```

Comment:
The ALL BOLD is a little hard on the eyes IMO. Maybe this?

```suggestion
            <p class="text-center">✨ <a href="#migrate"><span class="font-bold">NEW</span>
  Contact us and we'll migrate your Terraform to a modern language</a>. ✨</p>
```